### PR TITLE
Add Slovak translations 

### PR DESCRIPTION
**Pull request by mostypc123**
**Summary**

I added a translation to slovak.

**Translation checklist**

- [x] I have ran the application to make sure my code runs
- [x] I have used the correct [ISO 639-1 code](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm) for my translations (e.g. `en`)
- [x] I have formatted all changes as described in the [contribution style guide](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide)
- [x] I have added the language to `supportedLanguages` in alphabetical order
- [x] I have added the language to the table in [README.md](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md#translations) in alphabetical order

### CODE REVIEW

**Comment by mostypc123** [comment]
@brentvollebregt please review the pr

**Comment by brentvollebregt** [comment]
Apologies for the delay - I don't have something I can review this properly right now but it looks like the prettier formatting check failed. 

Other than that, this is looking ok from what I can see currently👍

**Comment by mostypc123** [comment]
idk why but i formatted it two times with prettier, or do i need more config after ibstalling the vs code extension

**Comment by brentvollebregt** [comment]
What are you using to format? I used `npx prettier@2.8.8 --write .` on your branch as described in the contribution guidelines and it fixed the formatting issues.

**Comment by mostypc123** [comment]
i used the vscode extension

**Comment by brentvollebregt** [comment]
Please use `npx prettier@2.8.8 --write .` as described in the [contribution style guide](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide) (on the PR checklist)

**Comment by brentvollebregt** [comment]
I have the changes locally now - I'll push them for you

**Comment by mostypc123** [comment]
thanks

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -2291,6 +2359,10 @@ const supportedLanguages = [
     name: 'Italian (Italiano)',
     code: 'it',
   },
+  {
+    name: 'Slovak (Slovenčina)',
+    code: 'sk',
+  },
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2365]: You have checked " I have added the language to `supportedLanguages` in alphabetical order" but this is not. It should be under Serbian.

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -342,6 +352,7 @@ const translationMap = {
         ar: `خيارات محددة ${applicationName}`,
         fa: `گزینه‌های ویژه ${applicationName}`,
         sr: 'Precizno podešavanje auto-py-to-exe-a',
+        sk: 'Špecifické nastavenia ${applicationName}',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:355]: The variable is not being inserted correctly

```suggestion
        sk: `Špecifické nastavenia ${applicationName}`,
```

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -1284,6 +1322,10 @@ const translationMap = {
           'Budi oprezan kada koristiš dodatne datoteke u režimu jedne datoteke;\n' +
           `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>pročitaj ovo (na engleskom)</a>\n` +
           'i ažuriraj svoj kod da bi radio sa PyInstaller.',
+        sk:
+          'Buďte opatrný pri pridávaní súborov v režime jedného súboru;\n' +
+          '<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>prečítajte si tento príspevok (stránka v angličtine)</a>\n' +
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1327]: The variable is not being inserted

```suggestion
         `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>prečítajte si tento príspevok (stránka v angličtine)</a>\n` +
```

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

**Comment by mostypc123** [comment]
will do the things later im on github mobile right now idk how to even use it😂

**Comment by mostypc123** [comment]
looks like i figured it out, it's fixed

File: auto_py_to_exe/web/js/i18n.js [hunk 4]
```diff
@@ -2315,6 +2383,10 @@ const supportedLanguages = [
     name: 'Serbian (Srpski)',
     code: 'sr',
   },
+  {
+    name: 'Slovak (Slovenčina)',
+    code: 'sk',
+  }
   {
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2390]: Looks like this will fail to parse as it's missing a comma

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
Thanks for the contribution! This has been released in [v2.45.1](https://github.com/brentvollebregt/auto-py-to-exe/releases/tag/v2.45.1) 🚀

**Pull request merged**